examples/vision/cutmix.py

```diff
@@ -0,0 +1,383 @@
+"""
+Title: CutMix Augmentation for Image Classification
+Author: [Sayan Nath](https://twitter.com/SayanNa20204009)
+Date created: 2021/03/25
+Last modified: 2021/03/25
```

Comment:
These dates may need to be updated before you finish 👍

Reply:
Yes! I will change it when I will place my commit.